src/app/utils/validators/validators.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { FormControl } from '@angular/forms';
+import { listContainsValidator } from './list-contains.validator';
+
+describe('Validators', () => {
+    describe('listContainsValidators', () => {
+        it('listContainsValidator should not return null if it fails', () => {
+            const source = [
+                { publicKeyHex: '0x8C2CBcBE93b96ef66F854407548edE5f74f4c016ybdybdjdbdjdyydydjdjdjdjdd' }
```

Comment:
You can leave this as it is, but do you know you can put here whatever you want? It doesn't need to be publicKeyHex.

Reply:
Yeah, I know. I felt it would be better to test with what we are already using. I could use other keys

Reply:
But we do not have any wrapper for those validators. There are few our own validators and each have it's own file with testing. We can in fact create a class for example:
```
  class SbValidators {
    static listContains() {}
    static someOtherMethod
    ....
  }
```
That way this describe wrapper have more sense. But this is topic for a different discussion.